pkg/engine/lookup_func.go

```diff
@@ -99,7 +99,7 @@ func getDynamicClientOnKind(apiversion string, kind string, config *rest.Config)
 	return res, apiRes.Namespaced, nil
 }
 
-func getAPIReourceForGVK(gvk schema.GroupVersionKind, config *rest.Config) (metav1.APIResource, error) {
+func getAPIResourceForGVK(gvk schema.GroupVersionKind, config *rest.Config) (metav1.APIResource, error) {
```

Comment:
It's likely this is an API break which I'll need to drop. This function appears to be new (d564d4bf2d6c6a9781439db231c4aaa1d7a916f8) since the previous round: #6752 (v3.1.0?)

Reply:
it's a private function. Not an API break.